index.bs

```diff
@@ -2055,86 +2055,98 @@ for the APIs that they design.
+can still be viewed in newer user agents,
+and also that authors have a shallower learning curve.
+
+To support this,
+web specification writers need to describe how to interpret
+poorly-formed markup,
+as well as well-formed markup.
 
-When writing specifications for the web,
-be sure to note authoring requirements
-for well-formed markup (the "conforming language"),
-and also
-implementation requirements
-for handling poorly-formed markup (the "supported language").
+Implementers need to be able to understand the "supported language",
+which is more complex than the "conforming language"
+which authors should be aiming to use.
 
-This allows us to
-have a relatively clean and understandable language for authors,
-while at the same time supporting existing documents
-that make use of older or nonstandard constructs,
-and enabling better interoperability in error handling.
+<div class="example">
+For example,
+the [Processing model](https://html.spec.whatwg.org/multipage/tables.html#table-processing-model)
+for the `<table>` element
+explains how to process the contents of a `<table>` element,
+including cases where the contents do not conform
+to the [Content model](https://html.spec.whatwg.org/multipage/tables.html#the-table-element:concept-element-content-model).
+</div>
 
 <h3 id="avoid-ambiguity">Specify completely and avoid ambiguity</h3>
 <!-- was "Well-defined behavior" in the HTML Design Principles -->
 
-If the specification leaves room for implementations to make different choices
-which mean authors need to write different markup for different implementations,
-this adds a maintenance burden for authors.
-
-The specification should be complete enough
-for implementations to be interoperable,
-without depending on or referring to the details of other implementations.
-When there isn't a good reason to allow variation between implementations,
-it is important that things be specified unambiguously,
-so that implementations interoperate
-and web content doesn't end up
-depending on the [=implementation-defined=] behavior
-in one implementation and failing in another
-(which can cause higher development costs to fix the content for all browsers,
-or costs to users of broken content if it wasn't detected).
-This way,
-it is easier to author content
-that works in a variety of user agents.
-However,
-implementations should still be free
-to make improvements
-in areas such as user interface
-and quality of rendering.
+When specifying how a feature should work,
+make sure that there is enough information
+so that authors don't have to
+write different code to work with different implementations.
+
+If a specification isn't specific enough,
+implementers might make different choices
+which force authors to write extra code to handle the differences.
+
+Implementers shouldn't need to check details of other implementations
+to avoid this situation.
+Instead, the specification should be complete and clear enough
+on its own.
+
```

Comment:
Possibly add a note to implementers here to file issues on a spec when the necessary level or clarity isn't present